backend/controllers/allocated.js

```diff
@@ -59,7 +60,7 @@ const allocated = async (req, res) => {
   for (const row of allocatedData) {
     const kw = keyworkerData.find(i => i.staffId === row.staffId);
     if (kw) {
-      row.keyworkerDisplay = `${kw.lastName}, ${kw.firstName}`;
```

Comment:
Actually I was thinking of changing this to return the raw first & last names and doing all the formatting for display in the frontend :(